.circleci/config.yml

```diff
@@ -235,7 +269,9 @@ jobs:
           key: amplify-verdaccio-cache-{{ .Branch }}-{{ .Revision }}
       - node/install:
           install-yarn: true
-          node-version: '16.13'
+          node-version: '18.15'
+      - run: yarn config set workspaces-experimental true
+      - run: yarn cache clean --force
```

Comment:
Why the change to workspaces and cache here?

Reply:
The updated dependencies were not being reflected due to cache for the Android container. This causes the `build-app-android-test` job to fail. Refer [error logs in this failed run of the test](https://app.circleci.com/pipelines/github/aws-amplify/amplify-codegen/2887/workflows/63eec25c-45bb-4be0-b72e-8b74ab59bd92/jobs/15860).